src/main/java/org/gitlab4j/api/PersonalAccessTokenApi.java

```diff
@@ -0,0 +1,49 @@
+package org.gitlab4j.api;
+
+import jakarta.ws.rs.core.Response;
+import org.gitlab4j.api.models.PersonalAccessToken;
+import org.gitlab4j.api.utils.ISO8601;
+
+import java.util.Date;
+
+/**
+ * This class provides an entry point to all the GitLab API personal access token calls.
+ *
+ * @see <a href="https://docs.gitlab.com/ce/api/personal_access_tokens.html">Personal access token API at GitLab</a>
+ */
+public class PersonalAccessTokenApi extends AbstractApi {
+
+    public PersonalAccessTokenApi(GitLabApi gitLabApi) {
+        super(gitLabApi);
+    }
+
+
+    /**
+     * Rotates the given personal access token.
+     * The token is revoked and a new one which will expire in one week is created to replace it.
+     * Only working with GitLab 16.0 and above.
+     *
+     * @return the newly created PersonalAccessToken.
+     * @throws GitLabApiException if any exception occurs
+     */
+    public PersonalAccessToken rotatePersonalAccessToken() throws GitLabApiException {
+        return rotatePersonalAccessToken(null);
+    }
+
+    /**
+     * Rotates the given personal access token.
+     * The token is revoked and a new one which will expire in one week is created to replace it.
+     * Only working with GitLab 16.0 and above.
+     *
```

Comment:
Usually we mention the endpoint called in the javadoc (easier to find later), can you add something like:

```
     * <pre><code>GitLab Endpoint: POST /personal_access_tokens/self/rotate</code></pre>
```